assets/js/components/Work/Tabs/Administrative/Administrative.jsx

```diff
@@ -197,33 +206,152 @@ const WorkTabsAdministrative = ({ work }) => {
                     label="Project Cycle"
                     defaultValue={projectCycle}
                   />
-                ) : (
-                  <p>{projectCycle}</p>
-                )}
+                ) : projectCycle ? (
+                  <Button
+                    isText
+                    data-testid="facet-link"
+                    className="break-word"
+                    onClick={() =>
+                      handleFacetLinkClick("ProjectCycle", projectCycle)
+                    }
+                    css={{ padding: "0", textTransform: "none !important" }}
+                  >
+                    <span>{projectCycle}</span>
+                  </Button>
+                ) : null}
+              </UIFormField>
+
+              <UIFormField label="Project / Job Name">
+                {isEditing ? (
+                  <UIFormInput
+                    data-testid="project-name"
+                    isReactHookForm
+                    placeholder="Project Name"
+                    name="projectName"
+                    label="Project Name"
+                    defaultValue={projectName}
+                  />
+                ) : projectName.length > 0 ? (
+                  <Button
+                    isText
+                    data-testid="facet-link"
```

Comment:
Same here?   Matchers in a single test file might get confused with multiple same named `data-testid`s, unless you're purposefully trying to grab a group of 'em?